### PR TITLE
Actually perform a HEAD request when validating a SSO token

### PR DESCRIPTION
**Pull request by daehn**
## What's in this PR?

* Actually perform a HEAD request when validating a SSO token.

### CODE REVIEW

File: Source/Registration/Company/CompanyLoginRequester.swift [hunk 1]
```diff
@@ -47,8 +47,8 @@ extension URLQueryItem {
 }
 
 @objc public protocol URLSessionProtocol: class {
-    @objc(dataTaskWithURL:completionHandler:)
-    func dataTask(with url: URL, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask
+    @objc(dataTaskWithRequest:completionHandler:)
+    func dataTask(with request: URLRequest, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Swift.Void) -> URLSessionDataTask
```
Comment (alexaubry) [inline Source/Registration/Company/CompanyLoginRequester.swift]: Why do we need `Swift.Void`?

Comment (daehn) [reply Source/Registration/Company/CompanyLoginRequester.swift]: We don't, just copied and pasted from the `URLSession.swift` header.

Comment (daehn) [reply Source/Registration/Company/CompanyLoginRequester.swift]: @alexaubry updated!

**Comment by daehn** [comment]
@zenkins release patch

**Pull request merged**